.github/workflows/deploy.yml

```
@@ -36,15 +36,35 @@ on:
           - web
           - good-job
         default: all
+      git_tag:
+        description: Git tag to deploy (optional). Will use workflow ref if not provided.
```

For clarity, are we able to insert the ref name here?

```suggestion
        description: Git tag to deploy (optional). Will use the reference ${{ github.ref_name}} if not provided.
```

---

Or maybe use the `default` option:

```suggestion
        description: Git tag to deploy (optional). Will use workflow ref if not provided.
        default: ${{ github.ref_name }}
```